WebDriverAgentTests/IntegrationTests/XCUIElementFBFindTests.m

```diff
@@ -298,18 +280,13 @@ - (void)testDescendantsWithClassChainWithIndex
 {
   NSArray<XCUIElement *> *matchingSnapshots;
   NSString *queryString = @"XCUIElementTypeWindow/*/*[2]/*/*/XCUIElementTypeButton";
-  if (@available(iOS 13.0, *)) {
-    // iPhone
-    queryString = @"XCUIElementTypeWindow/*/*/*/*[2]/*/*/XCUIElementTypeButton";
-    matchingSnapshots = [self.testedApplication fb_descendantsMatchingClassChain:queryString
-                                                     shouldReturnAfterFirstMatch:NO];
-    if (matchingSnapshots.count == 0) {
-      // iPad
-      queryString = @"XCUIElementTypeWindow/*/*/*/*/*[2]/*/*/XCUIElementTypeButton";
-      matchingSnapshots = [self.testedApplication fb_descendantsMatchingClassChain:queryString
-                                                       shouldReturnAfterFirstMatch:NO];
-    }
-  } else {
+  // iPhone
+  queryString = @"XCUIElementTypeWindow/*/*/*/*[2]/*/*/XCUIElementTypeButton";
```

Comment:
do we need now the above expression that initializes queryString?

Reply:
oh, yea. will modify here